### PR TITLE
swapping model IDs in the results

### PR DESCRIPTION
**Pull request by andrewkern**
edits meant to deal with https://github.com/popsim-consortium/manuscript/issues/42

### CODE REVIEW

**Comment by andrewkern** [comment]
hey @asiepel if you are looking at the manuscript now i'm looking for people to review this PR...

**Comment by asiepel** [comment]
gotta quit now (almost 1 here) but can look in the am if you’d like

On Dec 20, 2019, at 12:45 AM, Andrew Kern <notifications@github.com<mailto:notifications@github.com>> wrote:


hey @asiepel<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_asiepel&d=DwMCaQ&c=mkpgQs82XaCKIwNV8b32dmVOmERqJe4bBOtF0CetP9Y&r=rNAqNCgksGfltoLkih1npA&m=D-85ZgttAZwXiajikzgP2oIB2Vc97-CCOOIZ0wCAXPU&s=tQfVxDh2_O6K-1wCRgRLCaCQNg7UAKmNhmnVOBkZseU&e=> if you are looking at the manuscript now i'm looking for people to review this PR...

**Comment by andrewkern** [comment]
sure yeah-- just trying to get eyes on this thing.

File: popSimManu.tex [hunk 1]
```diff
@@ -43,6 +43,13 @@
 \makeatother
 \usepackage[utf8]{inputenc}
 
+\usepackage{fancyhdr}
```
Comment (andrewkern) [inline popSimManu.tex:47]: I added a fancy header to the document. I like how it looks- others?

**Review by grahamgower — APPROVED** [review]
Edits look good. I like the fancy header too.

**Comment by petrelharp** [comment]
I rebased this.

**Comment by andrewkern** [comment]
i'm ready to merge if you are cool with the edits

**Comment by petrelharp** [comment]
Looks good to me. Here's a latexdiff to make reviewing easy.
[popSimManu-diffd810bbabcac79f96601a52d7f5563c57eea42edc.pdf](https://github.com/popsim-consortium/manuscript/files/3988667/popSimManu-diffd810bbabcac79f96601a52d7f5563c57eea42edc.pdf)

**Comment by dschride** [comment]
Late to this party. I was going to suggest that we could follow up each usage of a model ID with a /citep of the paper; we do this in some cases but not all. This would provide a hint to any readers who might have forgotten the definition of the ID (which will probably be most of them) and perhaps save them a trip back to the table.

**Comment by andrewkern** [comment]
not sure. jerome's original comment was that we were citing the papers over and over.

**Comment by dschride** [comment]
Yes that is true, but when looking through the new version I just thought that could potentially be an issue for some readers, and rather than describing the model each time perhaps showing the label and citing the paper would be kind of a compromise. But I suppose we can leave it as is and see how it goes down with reviewers.

**Pull request merged**